urdf/include/urdf/model.h

```diff
@@ -56,6 +57,7 @@ class Model: public ModelInterface
   bool initFile(const std::string& filename);
   /// \brief Load Model given the name of a parameter on the parameter server
   bool initParam(const std::string& param);
+  bool initParamWithNodeHandle(const std::string& param, const ros::NodeHandle& nh);
```

Comment:
I see what you mean, yeah.  This sequence of events is unfortunate :(.  Ideally, we'd actually kill off `initParamWithNodeHandle()` completely, and just add the default-initialized argument to `initParam`.  We could do that in this PR, but at that point we have Indigo and Kinetic heading off in different directions, which would also suck.

So with all of that in mind (and with what @sloretz said), I guess this is fine as it is.